### PR TITLE
Pr: localize views - mailers

### PR DESCRIPTION
**Pull request by pavanjoshi914**
assignment mailer
groups mailer
pending invitation mailer
user mailer
correcting duplication of keys

Fixes #

### Screenshots of the changes (If any) -

* Group Mailer -

create group

image

add member

image

* Assignment mailer

create assignment

image

update assignment

image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Review by tachyons — APPROVED** [review]

File: config/locales/views/mailer/hi.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+en:
+  mailer:
+    welcome_heading: "सर्किटवर्स में आपका स्वागत है, %{name}"
+    keep_building: "निर्माण करते रहे"
+    view_project: "परियोजना देखें"
```
Comment (Shivansh2407) [inline config/locales/views/mailer/hi.yml:5]: I think we should make use of प्रोजेक्ट instead of परियोजना, to make it less complex.

File: config/locales/views/mailer/hi.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  group_mailer:
+    group_created_html: "आपने सफलतापूर्वक एक नया समूह बना लिया है, <span class='mailer-highlighted-text'> %{name} </span>।"
+    view_group: "समूह देखें"
+    member_added_to_group_html: "आपको <span class='mailer-highlighted-text'>%{group_name}</span> नाम के एक नए समूह में जोड़ा गया है। <br>
+    समूह का मार्गदर्शन <span class='mailer-highlighted-text'>%{user_name}</span> द्वारा किया जाता है।"
+  assignment_mailer:
+    assignment_created_html: "<span class='mailer-highlighted-text'>%{name} (%{deadline})</span> नाम का एक की नयी असाइनमेंट <span class='mailer-highlighted-text'> %{group_name} में असाइन की की गयी है </span>. <br>
```
Comment (Shivansh2407) [inline config/locales/views/mailer/hi.yml:12]: Please check the sentence structure again and update this. It has multiple same words like की की and grammatical mistakes.

File: config/locales/views/mailer/hi.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  user_mailer:
+    custom_email:
+      hello_user: "नमस्ते, %{name}"
+    featured_circuit_email:
+      circuit_featured_html: "वाह! आपकी परियोजना, <span class='mailer-highlighted-text'>%{name}</span> मंच पर प्रदर्शित कि गयी है।"
+    forked_project_email:
+      project_forked_html: "आपने <span class='mailer-highlighted-text'>%{original_author}</span> से एक नयी परियोजना सफलतापूर्वक फोर्क कि है। <br>
+      आपके फोर्क किए गए परियोजना का नाम <span class='mailer-highlighted-text'>%{forked_name}</span> है और परियोजना का एक्सेस टाइप <span class='mailer-highlighted-text'>%{access_type}</span > है।"
+    new_project_email:
+      project_created_html: "आपने सफलतापूर्वक नयी परियोजना बना लि है, <span class='mailer-highlighted-text'>%{name}</span>।"
```
Comment (Shivansh2407) [inline config/locales/views/mailer/hi.yml:26]: लि -> ली

**Review by Shivansh2407 — CHANGES_REQUESTED** [review]
Nice work @pavanjoshi914 :+1: I have suggested a few changes here, please update them and we'll be good to go! Thanks :tada:

File: config/locales/views/mailer/hi.yml (hunk 1 above)
Comment (Shivansh2407) [reply config/locales/views/mailer/hi.yml:5]: We can replace परियोजना to प्रोजेक्ट everywhere in the file.

Comment (pavanjoshi914) [reply config/locales/views/mailer/hi.yml:5]: I have used परियोजना from starting in other modules also as suggested by @ShreyaPrasad1209 shall I change this everywhere?

Comment (pavanjoshi914) [reply config/locales/views/mailer/hi.yml:5]: @ShreyaPrasad1209 and me decided to use परियोजना earlier. so I used this in other modules too, shall I change this everywhere or only for mailers.

if the change is required I will create new pr as change will push different files for other modules too..

Comment (Shivansh2407) [reply config/locales/views/mailer/hi.yml:5]: @pavanjoshi914 let it be परियोजना for now, because we would have to change it in other files as well.

File: config/locales/views/mailer/hi.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+en:
+  mailer:
+    welcome_heading: "सर्किटवर्स में आपका स्वागत है, %{name}"
```
Comment (ShreyaPrasad1209) [inline config/locales/views/mailer/hi.yml:3]: Can this name be converted to Hindi? It’s showing I’m English even in Hindi translations

File: config/locales/views/mailer/hi.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    assignment_created_html: "<span class='mailer-highlighted-text'>%{name} (%{deadline})</span> नाम का एक की नयी असाइनमेंट <span class='mailer-highlighted-text'> %{group_name} में असाइन की की गयी है </span>. <br>
+    असाइनमेंट का मार्गदर्शन <span class='mailer-highlighted-text'>%{mentor_name}</span> द्वारा किया जाता है"
+    view_assignment: "असाइनमेंट देखें"
+    assignment_updated_html: "असाइनमेंट <span class='mailer-highlighted-text'>%{name} (%{deadline})</span> को अपडेट कर दिया गया है। <br>
+    असाइनमेंट का मार्गदर्शन <span class='mailer-highlighted-text'>%{mentor_name}</span> द्वारा किया जाता है"
+  user_mailer:
+    custom_email:
+      hello_user: "नमस्ते, %{name}"
+    featured_circuit_email:
+      circuit_featured_html: "वाह! आपकी परियोजना, <span class='mailer-highlighted-text'>%{name}</span> मंच पर प्रदर्शित कि गयी है।"
```
Comment (ShreyaPrasad1209) [inline config/locales/views/mailer/hi.yml:21]: Here it should be badi e in “ कि” instead of choti e ki matra

File: config/locales/views/mailer/hi.yml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    forked_project_email:
+      project_forked_html: "आपने <span class='mailer-highlighted-text'>%{original_author}</span> से एक नयी परियोजना सफलतापूर्वक फोर्क कि है। <br>
```
Comment (ShreyaPrasad1209) [inline config/locales/views/mailer/hi.yml:23]: Same issue of “ कि” here also

File: config/locales/views/mailer/hi.yml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+      आपके फोर्क किए गए परियोजना का नाम <span class='mailer-highlighted-text'>%{forked_name}</span> है और परियोजना का एक्सेस टाइप <span class='mailer-highlighted-text'>%{access_type}</span > है।"
+    new_project_email:
+      project_created_html: "आपने सफलतापूर्वक नयी परियोजना बना लि है, <span class='mailer-highlighted-text'>%{name}</span>।"
+    welcome_email:
+      know_more: "अधिक जानने के लिए"
+      welcome_text_html: "साइन अप करने के लिए धन्यवाद।<br> सर्किटवर्स के साथ आप साधारण गेट से लेकर काम्प्लेक्स सेक्वेंशिअल सर्किट्स, प्लॉट टाइमिंग डायग्राम, आटोमेटिक सर्किट जनरेशन, स्टैण्डर्ड आईसी के बारे में अधिक, और बहुत कुछ जानिए, आप सर्किटवर्स के साथ मुफ्त में डिजिटल लॉजिक सर्किट डिजाइन और बहुत कुछ कर सकते हैं!"
```
Comment (ShreyaPrasad1209) [inline config/locales/views/mailer/hi.yml:29]: After “dhyanwad” maybe add an exclamation mark!

File: config/locales/views/mailer/hi.yml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+      design_heading: "डिज़ाइन करना"
+      design_description: "क्या आपको पता नहीं कैसे डिजिटल सर्किट डिजाइन करना है? सर्किटवर्स, शुरुआती लोगों के लिए उपयोग करना आसान है। सर्किटवर्स डॉक्यूमेंटेशन आपको जल्दी से शुरू करने में मदद करेगा।"
+      learn_heading: "सीखना"
+      learn_description: "क्या आप डिजिटल सर्किट के बारे में अधिक जानना चाहते हैं? सर्किटवर्स इंटरैक्टिव बुक आपको डिजिटल सर्किट कैसे काम करती है, इसकी पूरी समझ देती है।"
```
Comment (ShreyaPrasad1209) [inline config/locales/views/mailer/hi.yml:33]: Rephrase this sentence to - “Digital circuits kaise kaam karti h, iski poori samjh CircuitVerse interactive book deti hai. “

File: config/locales/views/mailer/hi.yml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+  pending_invitation_email:
+    main_heading: "सर्किटवर्स में आपका स्वागत है"
+    main_description_html: "आपको %{user_name} द्वारा %{name} नाम के एक नए समूह में जोड़ा गया है।<br><br>
+     डिजिटल सर्किट की दुनिया को एक्स्प्लोर करने के लिए सर्किटवर्स के साथ साइन अप करें। एक बार जब आप सर्किटवर्स के साथ साइन अप कर लेते हैं तो आप समूह का उपयोग कर सकते हैं।"
```
Comment (ShreyaPrasad1209) [inline config/locales/views/mailer/hi.yml:37]: Maybe rephrase this to - digital circuit ki duniya mei khoj karne ke liye” keeping rest of the sentence same

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
@pavanjoshi914 everything looks good to me except for few things. Please have a look and let me know what you think!

**Comment by Me-Priyank** [comment]
UI completely redesigned since
